source/topics/cli/how-to-install-cli.md

```diff
@@ -1,10 +1,26 @@
-The Aptible CLI is packaged as a Ruby gem. To install it, you'll first need to make sure you have Ruby installed locally (version 1.9.2 or greater). To find out how to install Ruby on your system, refer to [the installation guide](https://www.ruby-lang.org/en/documentation/installation/).
+## Aptible Toolbelt
+The preferred method of installing the Aptible CLI is via [Aptible Toolbelt](/toolbelt/). The Toolbelt pins dependencies and will usually run much faster than using the gem.
+
+## Ruby Gem  
+The Aptible CLI is also packaged as a Ruby gem that you can use directly. To install it, you'll first need to make sure you have Ruby installed locally (version 1.9.2 or greater). To find out how to install Ruby on your system, refer to [the installation guide](https://www.ruby-lang.org/en/documentation/installation/).
```

Comment:
I'd say that if they don't have Ruby yet, we encourage them to use the Toolbelt instead (I *can* understand not wanting to install the Toolbelt if you already have a Ruby install for philosophical reasons, but if you don't even have a Ruby install, there really is no reason not to use it).